package.json

```diff
@@ -202,6 +203,8 @@
     "storybook": "SENTRY_UI_HOT_RELOAD='' start-storybook -p 9001 -c .storybook",
     "storybook-build": "build-storybook -c .storybook -o docs-ui/.storybook-out --quiet",
     "webpack-profile": "NO_TS_FORK=1 yarn -s webpack --profile --json > stats.json",
+    "serve-docs": "swagger-ui-watcher api-docs/openapi.json",
```

Comment:
the default here is `8000` so not sure if that's fine or if we would wanna set a specific port with `-p`